src/Components/Projects/Projects.tsx

```diff
@@ -1,7 +1,331 @@
-import React from 'react'
+// nadia.mohamed.taha166@gmail.com
+// @Password123!
+import axios from "axios";
+import { useContext, useEffect, useState } from "react";
+import NoData from "./../../Shared/NoData/NoData";
+import noData from "./../../assets/images/no-data.png";
+import Modal from "react-bootstrap/Modal";
+import { useNavigate } from "react-router-dom";
+import { AuthContext } from "./../../Context/AuthContext";
+import { ToastContext } from "../../Context/ToastContext";
+import { useForm } from "react-hook-form";
 
-export default function Projects() {
+const Projects: React.FC = () => {
+  const { baseUrl, requestHeaders }: any = useContext(AuthContext);
+  const { getToastValue }: any = useContext(ToastContext);
+  const [project, setProject] = useState({});
+  const [projectDetails, setProjectDetails] = useState({});
+  const [projects, setProjects] = useState([]);
+  const navigate = useNavigate();
+  let [itemId, setItemId]: any = useState(0);
+  const [isLoading, setIsLoading] = useState(false);
+
+  interface FormValues {
+    title: any;
+    description: string;
+  }
+  const {
+    register,
+    setValue,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FormValues>();
+  // **********to use more than one modal in same component**********
+  const [modalState, setModalState] = useState("close");
+  // ********to close modal*******************
+  const handleClose = () => setModalState("close");
+  // ********to show view modal*******************
+  const showViewModal = (id: any) => {
+    console.log("proj", project);
```

Comment:
logs should be removed before merging the branch to the master, to avoid logging sensitive data to the users.